drivers/hmis/spec/support/ce_system_test_helper.rb

```diff
@@ -0,0 +1,92 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../requests/hmis/login_and_permissions'
+
+# Shared context for CE system tests.
+# These tests are tightly coupled to the workflows defined in CeWorkflows::Ac::WorkflowBuilder.
+RSpec.shared_context 'ce system test helper' do
+  include_context 'hmis base setup'
+
+  let!(:ds1) do
+    ds1 = GrdaWarehouse::DataSource.find_or_create_by!(hmis: 'localhost', source_type: :sftp, name: 'HMIS', short_name: 'HMIS')
+
+    # This is slow because it creates the entire workflow for every test.
+    # However, because of test cleanup, it's easier to do this here than in a before_all block.
+    HmisUtil::JsonForms.new(env_key: 'allegheny', override_generate_cdeds_in_test: true).seed_record_form_definitions(roles: [:CE_REFERRAL_STEP, :ENROLLMENT]) # Seed enrollment form so it collects units
+    CeWorkflows::Shared::CeBuilderUtils.create_state_machine_custom_statuses(ds1)
+    workflow_builder = CeWorkflows::Ac::WorkflowBuilder.new(ds1)
+    workflow_builder.build_housing_workflow
+    workflow_builder.build_admin_assign_workflow
+
+    ds1
+  end
+
+  let!(:target_project) { create(:hmis_hud_project, data_source: ds1, ProjectType: 3) } # PSH
+  let!(:coc1) { create :hmis_hud_project_coc, data_source: ds1, project: target_project, coc_code: 'CO-500' }
```

Comment:
If we don't care what the coc code is, we can use this attribute to specify generating a coc
```suggestion
  let!(:target_project) { create(:hmis_hud_project, data_source: ds1, ProjectType: 3, with_coc: true) } # PSH
```